src/rules/text-transform.ts

```diff
@@ -0,0 +1,11 @@
+import { makeRules } from "../utils";
+import { RuleFn } from "./RuleConfig";
+
+// https://github.com/tachyons-css/tachyons/blob/master/src/_text-transform.css
+export const textTransformRules: RuleFn = () =>
```

Comment:
Nice! Can you include this from the `generate.ts` file so that people get it for free?

Reply:
Sorry! Will add!